lib/sufia/arkivo.rb

```diff
@@ -0,0 +1,11 @@
+require 'sufia/arkivo/config'
```

Comment:
Yes, I did decide to throw it in `lib/` intentionally, but I'm happy to revisit that decision.  In short, the reason was that I thought it might be easier to extract if I had the Zotero stuff all sitting in the same spot in `lib/` without much knowledge of the app within which it is contained. I made that decision early on, but then as I developed, I very well could have lost sight of this as I sprinted towards the finish line.